### ISSUE TITLE
XBMC-style TV art support

### ISSUE THREAD

**Issue by Omertron** [YAMJ/yamj-v2#1482]
Original [issue 1483](https://code.google.com/p/moviejukebox/issues/detail?id=1483) created by Omertron on 2010-05-22T19:54:00.000Z:

Since I share my library with XBMC and YAMJ, my TV setups look something
like this:
- Better Off Ted
- Season 1
  - Better Off Ted - S01E01 - Pilot.mkv
  - Better Off Ted - S01E01 - Pilot.nfo
  - Better Off Ted - S01E01 - Pilot.tbn
  - ...
- Season 2
  - ...
- fanart.jpg
- poster.jpg
- tvshow.nfo

My problem is that when YAMJ picks up the art work for the show, it doesn't
grab 

&quot;Better Off Ted/poster.jpg&quot; or &quot;Better Off Ted/Better Off Ted.jpg&quot;

it grabs 

&quot;Better Off Ted/Season 1/Better Off Ted - S01E01 - Pilot.tbn&quot;, which is
basically just fanart.

If YAMJ could run up the directory structure one level to see if there's a
poster/fanart/whatever there, that would be fantastic!
